strawberry/experimental/pydantic/fields.py

```diff
@@ -1,11 +1,15 @@
+import builtins
 from decimal import Decimal
 from typing import Any, List, Optional, Type
 from uuid import UUID
 
 import pydantic
+from pydantic import BaseModel
 from pydantic.typing import is_new_type, new_type_supertype
+from typing_extensions import Literal
 
-from .exceptions import UnsupportedTypeError
+from ...types.types import TypeDefinition
```

Comment:
Can you change this `from ...` import to an absolute import?

Reply:
oops, pycharm must have did that